### ISSUE TITLE
Font size adjustment for the whole website

### ISSUE THREAD

**Issue by Asovrix** [stalkerg/anime-pictures#82]
Some older users might probably remember that at least 4 or 5 years ago the website had smaller fonts overall.
You can kinda see it here for example: https://web.archive.org/web/20140701115015/http://anime-pictures.net:80/
Font size increase probably happened because of adapting the site to mobile web browsers?
The thing is, I didn't like such huge text too much, so I've made myself a userstyle which roughly switches it back to how it was before:
```css
body,b,a.big_tag {font-size:10pt}
.title,.sidebar_block h2 {font-size:10pt !important}
h1 {font-size:16pt;line-height:16pt}
input,select,.tags li a:not(.big_tag) {font-size:9pt}
.post_content,.sidebar_block,#search_form,#message_form,ul.body li,#chat_users {font-size:10pt;line-height:14pt}
.post_content_avatar {min-height:145px}
.post_content .body {padding:5px 0 5px 5px}
.menu {font-size:9pt;line-height:32px}
.menu select {font-size:7pt}
.menu,.menu ul li,.menu ul li a {height:32px}
.menu ul li {padding:0 8px}
.icon_add_image,.icon_new_message,.icon_new_image,.icon_delete_big,.icon_edit_big {width:24px;height:24px}
.form_table .icon_delete_big,.form_table .icon_edit_big,.icon_delete,.icon_frame,.icon_edit,.icon_close,.icon_apply {height:16px;width:16px}
.subscribed_tags p {line-height:24px}
#chat_field tr td {padding:0 2px}
```

Would it be possible to have some kind of way of changing the font size on the site, either in the form of increasing/decreasing it directly with -/+ buttons placed somewhere on the site (header/footer/user settings) or as alternative versions of already existing styles?

**Comment by stalkerg** [comment]
Yes, mobile and I started use monitor with 1.29 DPI (WQHD) in that case it's also important. 
What screen do you use usually?

**Comment by Asovrix** [comment]
I have 2 displays with different pixel densities. Both are 24'', but one is 1920x1200 and the other one is 2560x1440. The thing is I bought the WQHD monitor only in 2018.
No display scaling is involved anywhere, everything is set to 100%.

Later after I come back from work I can post screenshots of website with and without my style later and how CSS shows up for me.

**Comment by Asovrix** [comment]
When I compared CSS for current version and an old one from 2013, the font size for most elements is indeed mainly 16px/12pt.
Here are the differences I found:
- Main font used on the site: back then it was Trebuchet MS, while now it's Open Sans.
Letters in Open Sans appear to be slightly larger compared to Trebuchet MS.
**Trebuchet MS 12pt normal**
![notepad_2018-11-30_16-12-01](https://user-images.githubusercontent.com/44006686/49297373-e94c8280-f4ba-11e8-860d-8dd8ac1ca26c.png)
**Open Sans 12pt normal**
![notepad_2018-11-30_16-12-11](https://user-images.githubusercontent.com/44006686/49297439-0ed98c00-f4bb-11e8-94e1-ca1af320f0f5.png)
This is a big visual change when you put it on a website.

- Object/reference tags had a size of 12px, while now they are 10pt. The height of the li element hasn't changed at all though.
- Line-height is now adjusted only to specific elements, when before it was set to 1.3 on _body_ element. This made some things appear vertically smaller before.
- Header is significantly bigger.

Well... I guess it's actually not the font size that is the problem for me.
